auth-web/src/components/auth/mixins/NextPageMixin.vue

```diff
@@ -50,7 +50,14 @@ export default class NextPageMixin extends Vue {
     switch (this.currentUser?.loginSource) {
       case LoginSource.IDIR:
       case LoginSource.BCROS:
```

Comment:
You will need to leave the return to SEARCH_BUSINESS for IDIR users.  We don't want to apply the new logic for staff.

Reply:
I get that, Thanks, good catch @sutherlanda